.github/workflows/nodejs.yml

```
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
+        node-version: [20.x]
```

Why was 18.x removed? It's used in `release.yml` later on
```suggestion
        node-version: [18.x, 20.x]
```

---

Fixed!